components/workspaces/base/server/config/default/kustomization.yaml

```diff
@@ -71,6 +71,43 @@ replacements:
       group: rbac.authorization.k8s.io
       kind: RoleBinding
       name: rest-api-server:usersignup-reader
+  - fieldPaths:
+    - subjects.0.namespace
+    options:
+      create: true
+    select:
+      group: rbac.authorization.k8s.io
+      kind: RoleBinding
+      name: rest-api-server:workspace-server-editor
```

Comment:
this part is not clear to me. Is it needed? should not be required

Reply:
I see this diff when I revert this part of the change:

```diff
--- /tmp/before.yaml	2024-07-25 11:38:53.719579916 -0500
+++ /tmp/after.yaml	2024-07-25 11:39:09.404385347 -0500
@@ -541,7 +541,7 @@ roleRef:
   name: workspaces-spacebinding-reader
 subjects:
 - kind: ServiceAccount
-  name: rest-api-server
+  name: workspaces-rest-api-server
   namespace: workspaces-system
 ---
 apiVersion: rbac.authorization.k8s.io/v1
@@ -555,7 +555,7 @@ roleRef:
   name: workspaces-usersignup-reader
 subjects:
 - kind: ServiceAccount
-  name: rest-api-server
+  name: workspaces-rest-api-server
   namespace: workspaces-system
 ---
 apiVersion: rbac.authorization.k8s.io/v1
@@ -591,7 +591,7 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: workspaces-rest-api-server
-  namespace: workspaces-system
+  namespace: system
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding

```

The subject namespace is wrong for the rolebinding `rest-api-server:workspace-server-editor`.  It's in the same namespace as the service account, so it turns out fine, but it's not semantically correct.

Reply:
got it. thanks for looking into it! Shouldn't `kustomize` substitute it?

Reply:
It should (and it's correct prior to this change), but this PR without this change causes it to revert back to `system`, not `workspaces-system`.  I can dig into this further, but I'd like to get this merged so external staging isn't on fire anymore.

Reply:
Fixed in 87e1ba1